dear_petition/petition/export/mapper.py

```diff
@@ -34,6 +36,9 @@ def map_petitioner(data, petition, extra={}):
     data["NamePetitioner"] = getattr(
         record, "label", None
     )  # load.py line 28 (label is set to name attr)
+    data["SSN"] = extra.get("ssn", None)
```

Comment:
The serializer requires these fields. Did we run into issues with these fields not being set?

Reply:
I think it's an issue with `pdf writer`. I am able to see `data["SSN"]` and `extra["ssn"]`.

Reply:
Ah, it's because the PDF has a bad key for it, ``SNN``. It should work with that.